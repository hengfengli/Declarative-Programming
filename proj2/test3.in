+---+---+---+
|3..|.1.|...|
|..6|98.|..3|
|...|..3|.52|
+---+---+---+
|...|.7.|261|
|..4|...|7..|
|172|.9.|...|
+---+---+---+
|91.|6..|...|
|6..|.39|5..|
|...|.2.|..7|
+---+---+---+
