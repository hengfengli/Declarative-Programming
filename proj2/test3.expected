+---+---+---+
|349|512|876|
|256|987|143|
|781|463|952|
+---+---+---+
|593|874|261|
|864|251|739|
|172|396|485|
+---+---+---+
|917|645|328|
|628|739|514|
|435|128|697|
+---+---+---+
